api/src/pcapi/core/providers/repository.py

```diff
@@ -62,3 +64,28 @@ def get_allocine_theater(venue: Venue) -> AllocineTheater:
 def is_venue_known_by_allocine(venue: Venue) -> bool:
     allocine_theater = get_allocine_theater(venue)
     return allocine_theater is not None
+
+
+def get_cinema_provider_pivot_for_venue(venue: Venue) -> CinemaProviderPivot:
```

Comment:
```suggestion
def get_cinema_provider_pivot_for_venue(venue: Venue) -> Optional[CinemaProviderPivot]:
```
`one_or_none()` peut retourner... `None` :)